pkg/crane/pull.go

```diff
@@ -34,12 +34,25 @@ func init() { Root.AddCommand(NewCmdPull()) }
 
 // NewCmdPull creates a new cobra.Command for the pull subcommand.
 func NewCmdPull() *cobra.Command {
-	return &cobra.Command{
+	cmd := &cobra.Command{
 		Use:   "pull",
```

Comment:
I think you can just update `Use` to achieve the same thing.

Reply:
right. I don't why I did that. Reverted it now.